src/languages/en.ts

```diff
@@ -127,6 +127,10 @@ type States = Record<keyof typeof COMMON_CONST.STATES, StateValue>;
 
 type AllCountries = Record<Country, string>;
 
+function integrationName(fallbackName: string, integration?: ConnectionName): string {
```

Comment:
I'm hesitant to add a function like this into the templates. I don't see a scenario where we would need to use the callback name since we control when we show this text. Can we just pass `CONST.POLICY.CONNECTIONS.NAME_USER_FRIENDLY[integration]` into the template directly?

Reply:
We use the same code a few times in this file
- https://github.com/Expensify/App/blob/b0947e029faea27eebf348d125032a992d6181c3/src/languages/en.ts#L2804
- https://github.com/Expensify/App/blob/b0947e029faea27eebf348d125032a992d6181c3/src/languages/en.ts#L2812
- https://github.com/Expensify/App/blob/b0947e029faea27eebf348d125032a992d6181c3/src/languages/en.ts#L2771

I thought that it would be worth it extracting the code as a separate function and use it everywhere. I also moved `noAccountsFound` and `noAccountsFoundDescription` to the `common` section so we have two more places where we use it.

If you don't like the idea I can move it back and revert changes. Please let me know 🙏

Reply:
Yeah I mean I don't think we'll ever need to use the generic copy i.e. "this integration" and the connection names are the same in Spanish vs English so I think it would be better to revert it back if that's okay